plenum/test/node_request/message_request/test_setup_for_non_master.py

```diff
@@ -0,0 +1,54 @@
+import time
+import pytest
+
+from plenum.common.messages.node_messages import PrePrepare
+from plenum.common.types import f
+from stp_core.common.log import getlogger
+from plenum.test.helper import sdk_send_random_and_check
+
+logger = getlogger()
+
+
+def test_setup_for_non_master(looper,
```

Comment:
Please add the description of what the test checks.

Reply:
done